src/test/java/com/microsoft/sqlserver/testframework/DBResultSet.java

```diff
@@ -157,8 +158,8 @@ public void verifydata(int ordinal,
         Object retrieved = this.getXXX(ordinal + 1, coercion);
 
         // Verify
-        //TODO: Check the intermittent verification error
-        //verifydata(ordinal, coercion, expectedData, retrieved);
+        // TODO: Check the intermittent verification error
+        // verifydata(ordinal, coercion, expectedData, retrieved);
```

Comment:
this is not related to this pr. we need to investigate this later for the intermittent bvt failures